transaction-kafka-samples/transaction-spring-data-processor/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+spring:
+    cloud.stream:
+        bindings:
+            input.destination: person-command
+            output.destination: person-event
+        kafka.binder.transaction:
+            transaction-id-prefix: person-
+            producer:
+              configuration:
+                retries: 1
```

Comment:
Should this be 1 or should it be higher?

Reply:
@bjartek I think `1` should be good if it worked.